pandas/core/series.py

```diff
@@ -5745,7 +5745,49 @@ def to_period(self, freq: str | None = None, copy: bool | None = None) -> Series
     _info_axis_name: Literal["index"] = "index"
 
     index = properties.AxisProperty(
-        axis=0, doc="The index (axis labels) of the Series."
+        axis=0,
+        doc="""
+        The index (axis labels) of the Series.
+        Get the index (row labels) of the Series.
+
+        Returns
+        -------
+        Index
+            The index labels of the Series.
+
+        See Also
+        --------
+            Series.reindex:
+                Conform Series to new index.
+            Series.set_index:
+                Set Series as DataFrame index.
+            Index:
+                The base pandas index type.
+
+        Notes
+        -----
+        The index of a Series is used to label and identify each element of the
+        underlying data. The index can be thought of as an immutable ordered set
+        (technically a multi-set, as it may contain duplicate labels), and is
+        used to index and align data in pandas.
+
+        For more information on pandas indexing, see the `indexing user guide
+        <https://pandas.pydata.org/docs/user_guide/indexing.html>`__.
+        Examples
```

Comment:
I don't think it is needed, but for consistency there should be a blank like here:

```suggestion

        Examples
```